instant-messenger: change Riot privacy conern link to #1049

Mikaela opened the pull request:
In case we decide to NOT merge #1047

Mikaela commented on _includes/sections/instant-messenger.html:
```diff
@@ -24,7 +24,7 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
 title="Riot.im"
 image="/assets/img/tools/Riot.png"
-description="Riot.im is a decentralized free-software chatting application based on the <a href\"https://matrix.org/\">Matrix</a> protocol, a recent open protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too. <a href=\"https://github.com/vector-im/riot-web/issues/6779\"><span class=\"badge badge-warning\" data-toggle= ...
+description="Riot.im is a decentralized free-software chatting application based on the <a href\"https://matrix.org/\">Matrix</a> protocol, a recent open protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too. <a href=\"https://github.com/vector-im/riot-web/issues/6779\"><span class=\"badge badge-warning\" data-toggle= ...
```
> Riot sends a lot of data to matrix.org and vector.im with default settings that aren't trivial to change, also with selfhosted homeservers

TODO: change this part to something actually reflecting the issue, like the most important problems. I think that would be in general metadata like with Wire and the removed messages never getting removed.

erciccione commented:
I tend to be in favour of this PR and against #1047. But i have to catch up on the discussions before giving a fully informed opinion.

Mikaela commented:
I have removed the description from this PR and the first commit as this appears to be more popular PR and in another commit I fixed the tooltip text that was leftover from the old link.

My personal preference would still be delisting Riot until they have gotten their privacy issues resolved and then listing them again.

nitrohorse commented:
> My personal preference would still be delisting Riot until they have gotten their privacy issues resolved and then listing them again.

I agree. Based on https://github.com/privacytoolsIO/privacytools.io/issues/1049 I think either we should remove Riot for now, but I'd be ok with bumping it to the "Worth Mentioning" section with the "privacy conerns" tooltip warning users.

Perelandra0x309 commented:
You are missing a closing element on your second anchor link:
<a href=\"https://github.com/privacytoolsIO/privacytools.io/issues/1049\**"<**span

Mikaela commented:
> I agree. Based on the issues listed in #1049 I think we should remove Riot for now.

Do I understand correctly that you are requesting changes to block merging of this and will be giving an approving review to https://github.com/privacytoolsIO/privacytools.io/pull/1047 ?

> We could also bump it to the "Worth Mentioning" section with the "privacy conerns" tooltip warning users but that wouldn’t be my first choice.

I disagree as Riot is not an instant messenger for people, but teams, and would be blocked by https://github.com/privacytoolsIO/privacytools.io/issues/1065.

> You are missing a closing element on your second anchor link:
> `<a href="https://github.com/privacytoolsIO/privacytools.io/issues/1049\"<span`

Could you clarify what is wrong in the file and what should it be instead? I looked at it a few times, but I don't spot the issue. I find the code too confusing :(

nitrohorse commented:
> Do I understand correctly that you are requesting changes to block merging of this and will be giving an approving review to #1047 ?

Yeah 👍🏼 I’ll make an approving review if 1047.

> I disagree as Riot is not an instant messenger for people, but teams, and would be blocked by #1065.

Ah I see, that makes sense.

Perelandra0x309 commented:
> Could you clarify what is wrong in the file and what should it be instead? I looked at it a few times, but I don't spot the issue. I find the code too confusing :(

I think it's actually an inherited error from the existing version, but the html should be this:

 <a href="https://github.com/privacytoolsIO/privacytools.io/issues/1049\"><span

Add a ">" before the "<span" so it is valid HTML.

JonahAragon commented on _includes/sections/instant-messenger.html:
```diff
@@ -24,7 +24,7 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
 title="Riot.im"
 image="/assets/img/tools/Riot.png"
-description="Riot.im is a decentralized free-software chatting application based on the <a href\"https://matrix.org/\">Matrix</a> protocol, a recent open protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too. <a href=\"https://github.com/vector-im/riot-web/issues/6779\"><span class=\"badge badge-warning\" data-toggle= ...
+description="Riot.im is a decentralized free-software chatting application based on the <a href\"https://matrix.org/\">Matrix</a> protocol, a recent open protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too. <a href=\"https://github.com/vector-im/riot-web/issues/6779\"><span class=\"badge badge-warning\" data-toggle= ...
```
```suggestion
description="Riot.im is a decentralized free-software chatting application based on the <a href\"https://matrix.org/\">Matrix</a> protocol, a recent open protocol for real-time communication offering E2E encryption. It can bridge other communications via others protocols such as IRC too. <a href=\"https://github.com/vector-im/riot-web/issues/6779\"><span class=\"badge badge-warning\" data-toggle=\"tooltip\" title=\"The end-to-end encryption is currently in beta and the mobile client states 'End-to-end encryption is in beta and may not be reliable. You should not yet trust it to secure data.'\">Experimental <i class=\"far fa-question-circle\"></i></a></span> <a href=\"https://github.com/privacytoolsIO/privacytools.io/issues/1049\"><span class=\"badge badge-danger\" data-toggle=\"tooltip\" title=\"Riot has a lot of privacy issues that are tracked in our metaissue #1049.\">Privacy concerns</span></a>"
```
